### PR TITLE
feature display item name

### PR DESCRIPTION
**Pull request by NuriyaAkh**
_For an example of how to fill this template out, [see this Pull Request](https://github.com/the-collab-lab/tcl-3-smart-shopping-list/pull/44)._

## Description
Feature to display item name in List.jsx, added Firebase configs 

## Acceptance Criteria

- The streamListItems function is used in App.jsx to get the items in the current list from the Firestore database
-  The list is passed into the List component as a prop named data
-  In List.jsx, the ListItem component is used to render the name of each item

## Type of Changes

|     | Type                       |
| --- | -------------------------- |
|     | :bug: Bug fix              |
| ✓   | :sparkles: New feature     |
|     | :hammer: Refactoring       |
|     | :100: Add tests            |
|     | :link: Update dependencies |
|     | :scroll: Docs              |

### CODE REVIEW

**Review by snowsneakers — COMMENTED** [review]
List items are showing! It is saying that there is a conflict in src/api/config.js

**Review by kaitlinLewis — APPROVED** [review]
Looks good!

File: src/views/Layout.jsx [hunk 1]
```diff
@@ -21,15 +21,24 @@ export function Layout() {
...
+					</a> */}
+					<NavLink to="/" className="Nav-link">
+						Home
+					</NavLink>
+					<NavLink to="/list" className="Nav-link">
+						List
+					</NavLink>
+					<NavLink to="/add-item" className="Nav-link">
+						Add Item
+					</NavLink>
```
Comment (cafloyd) [inline src/views/Layout.jsx:41]: Great job here, but let's make sure we're not doing extra work outside of the acceptance criteria listed on the ticket - this is something that Kaitlin and Patrick worked on this week! Please undo these changes to make sure we're not overwriting their work 🙏 

- [ ] revert changes related to navlinks

File: src/views/List.jsx [hunk 2]
```diff
@@ -7,11 +7,16 @@ export function List({ data }) {
...
-				 * TODO: write some JavaScript that renders the `data` array
-				 * using the `ListItem` component that's imported at the top
-				 * of this file.
-				 */}
+				{
+					/**
+					 * TODO: write some JavaScript that renders the `data` array
+					 * using the `ListItem` component that's imported at the top
+					 * of this file.
+					 */
```
Comment (cafloyd) [inline src/views/List.jsx:15]: Woohoo! This 'to-do' is now 'to-done'! Let's get rid of this comment to keep our code nice and tidy 😃 

- [ ] delete comments

**Review by cafloyd — CHANGES_REQUESTED** [review]
Looks really good @NuriyaAkh and @meera-ramesh19 !! I just make a couple of comments about some minor changes to be made, but over all really nice job getting Firebase set up! 👏 👏 👏

File: src/views/Layout.jsx [hunk 3]
```diff
@@ -21,15 +21,24 @@ export function Layout() {
...
 				</nav>
```
Comment (georgeoffley) [inline src/views/Layout.jsx:42]: I think these `<NavLink>` components are part of the other task that the other team is working on. Looking at their PR it has this same code. There might be a merge conflict with this code. If I am not understanding something, please let me know.

https://github.com/the-collab-lab/tcl-48-smart-shopping-list/pull/16/files

**Review by georgeoffley — CHANGES_REQUESTED** [review]

File: src/views/Layout.jsx (hunk 1 above)
Comment (NuriyaAkh) [reply src/views/Layout.jsx]: done

File: src/views/List.jsx (hunk 2 above)
Comment (NuriyaAkh) [reply src/views/List.jsx]: done

File: src/views/Layout.jsx (hunk 3 above)
Comment (NuriyaAkh) [reply src/views/Layout.jsx]: I updated our code

**Review by cafloyd — APPROVED** [review]
Beautiful, thanks for making those changes! 🤩

**Review by georgeoffley — APPROVED** [review]

**Comment by georgeoffley** [comment]
Great job, y'all! Thanks for making those changes!!! Approved!!!

**Review by daredtech — APPROVED** [review]
Great job on your first PR!
